src/sdk/pynni/nni/curvefitting_assessor/curvefitting_assessor.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
```

Comment:
could you add your paper referenced and update the README?

Reply:
Added README doc and submitted it to a new pr, pls go to: https://github.com/Microsoft/nni/pull/481